documentation/examples.md

```diff
@@ -123,6 +123,18 @@ D("example.tld", REG, DSP, ...
 ```
 {% endcode %}
 
+## Set default records modifiers ##
+{% code title="dnsconfig.js" %}
+```javascript
+DEFAULTS(
+    NAMESERVER_TTL('24h'),
+    DefaultTTL('12h'),
+    CF_PROXY_DEFAULT_OFF
+);
+```
+
```

Comment:
There's an `{% endcode %}` missing here.

Reply:
Ooops! Thanks for catching that!